rascal2/widgets/controls/view.py

```diff
@@ -0,0 +1,147 @@
+"""Widget for setting up the Controls class."""
+
+from enum import Enum
+
+from PyQt6 import QtGui, QtWidgets
+from RATapi.utils.enums import Procedures
+
+from rascal2.config import path_for
+from rascal2.widgets.controls.model import FitSettingsModel
+from rascal2.widgets.delegates import BoolDelegate, EnumDelegate
+
+
+class ControlsWidget(QtWidgets.QWidget):
+    """Widget for editing the Controls object."""
+
+    def __init__(self, presenter):
+        super().__init__()
+        self.presenter = presenter
+        init_procedure = self.presenter.model.controls.procedure  # to set to whatever initial procedure is saved
+
+        # create fit settings view and setup connection to model
+        self.fit_settings = QtWidgets.QTableView()
+        self.fit_settings_model = FitSettingsModel(self.presenter)
+        self.fit_settings.setModel(self.fit_settings_model)
+        self.set_procedure(init_procedure)
+
+        # aesthetics of fit settings table
+        self.fit_settings.horizontalHeader().setVisible(False)
+        self.fit_settings.horizontalHeader().setStretchLastSection(True)
+        self.fit_settings.verticalHeader().setVisible(True)
+        self.fit_settings.setShowGrid(False)
+        self.fit_settings.setSelectionMode(self.fit_settings.SelectionMode.SingleSelection)
+
+        # create run button
+        self.play_icon = QtGui.QIcon(path_for("play.png"))
+        self.stop_icon = QtGui.QIcon(path_for("stop.png"))
+        self.run_button = QtWidgets.QPushButton(icon=self.play_icon, text="Run")
+        self.run_button.setStyleSheet("background-color: green;")
+        self.run_button.setCheckable(True)
+        self.run_button.toggled.connect(self.toggle_run_button)
+
+        # create box containing chi-squared value
+        chi_box = QtWidgets.QHBoxLayout()
```

Comment:
It will be helpfully to name the chi_box,  procedure_selector and any others so its easier to tell what they are from the name maybe chi_layout, procedure_layout